src/OpenTelemetry/Batch.cs

```diff
@@ -64,6 +63,19 @@ internal Batch(CircularBuffer<T> circularBuffer, int maxSize)
     /// </summary>
     public long Count { get; }
 
+    /// <summary>
+    /// Create an instance of <see cref="Batch{T}"/> containing a single item.
+    /// </summary>
+    /// <param name="item">Item to store in the batch.</param>
+    /// <returns><see cref="Batch{T}"/> containing the supplied item.</returns>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
With inlining there really is no difference calling the static vs the ctor. Demo: https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAMIR8AB14AbGFADKMgG68wMXAG4aNYgGYmpBkIYBvGgzNNdAIWwYwACwA8zOgD4GAVVy8uAcwCSPDJc2JIAFACUxqbmMcQA7AxcMADuoQBEdjCSkhAMydCSACZp4RrUMQC+0WbVFgzWto7Obp7ePrIYNsoRUeUx5vH1NvZO5K7CsDYw6ZnZuflQRSVllTG1tToMk4UQXJIAngy4GFAcYBhDjQ4AKi61McmZsAzXDCB6tSZ9/QxiULwKKZbGDYHZ7Q7XAD8DF4GBg+AYAF5Ehxsisfv17uYANoAWRgGDsEEKfnEYXxhOJpLEkgA8mI+LtcCwAII+HywXBeBQwAKSbxtcIAXSxZk2zCQlxGtwMkzhoVesPh4VFvQx/RZUB8HAEPAAcqjJABRBAqBm8XYsa52KAQZJ+ABmBuyoSV+FKmm+6qYCSSySljlurrh7vRPwq3vMnvV3jhUGCkgDCphIZVXpiX0jZkJvGZbqRKfhYf6VXTGNVm1efhDSLcObzIYAhMXSxUgA